google/cloud/bigtable/metadata_update_policy.cc

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 #include "google/cloud/bigtable/metadata_update_policy.h"
-#include "google/cloud/bigtable/internal/make_unique.h"
+#include "google/cloud/internal/make_unique.h"
```

Comment:
It seems that `make_unique.h` file is included but never used, this inclusion can be avoided.

Reply:
Done.